.gitignore

```diff
@@ -102,6 +102,10 @@ ENV/
 .idea/
 .DS_Store
 .pytest_cache
+.mypy_cache
+
+# Test leftovers
+tmp-tslearn-test.txt
```

Comment:
No longer needed since you specify a new file name in the test, right?

Reply:
Ah, I updated the PR description accordingly. It is still required since executing the doctests still produces these files since for example `save_time_series_txt()` still uses them. For the propose of readability, I wouldn't touch that snippet of documentation.

Reply:
ok